src/codesyncd/utils.ts

```diff
@@ -241,3 +241,18 @@ export const getDiffsBeingProcessed = () => {
 export const setDiffsBeingProcessed = (diffs: Set<string>) => {
 	CodeSyncState.set(CODESYNC_STATES.DIFFS_BEING_PROCESSED, diffs);
 };
+
+export const getTabsBeingProcessed = () => {
+	const tabsInState = CodeSyncState.get(CODESYNC_STATES.TABS_BEING_PROCESSED);
+	return tabsInState || new Set();
+};
+
+export const setTabsBeingProcessed = (tabs: Set<string>) => {
+	CodeSyncState.set(CODESYNC_STATES.TABS_BEING_PROCESSED, tabs);
+};
+
+export const getRandomIndex = (length: number) => Math.floor( Math.random() * length );
+
+export const isRelative = (relative :string ) => {
```

Comment:
1. isRelative -> isRelativePath
2. argument should be `path` instead of `relative`

Reply:
Regarding 2. , I have renamed it to `relative_path` as `path` is a reserved keyword

Reply:
I meant to write `filePath`, how can we say it is a relativePath when the same function is checking this thing